src/timepicker/timepicker.component.ts

```diff
@@ -115,6 +115,11 @@ export class TimepickerComponent
   seconds: string;
   meridian: string;
 
+  // aria-label variables
+  labelHours = 'hours';
```

Comment:
`@Input` ?

Also add these new options to the timepicker config

Reply:
yes, should be configurable via config, and config should be applied before ngOnInit